src/vscode/commands/Repl.ts

```diff
@@ -66,28 +72,68 @@ export async function inlineEval(state: ExtensionState) {
     })
 }
 
-export async function attachRepl(state: ExtensionState, ctx: vscode.ExtensionContext) {
+export async function startReplAndAttach(state: ExtensionState, ctx: vscode.ExtensionContext) {
+    try {
+        const installedSlimeInfo = await installAndConfigureSlime(state)
+        const cwd = await getWorkspaceOrFilePath()
+        const cmd = vscode.workspace.getConfiguration('alive').swank.startupCommand
+        const attachToRunningRepl = (out: string) => {
+            swankOutputChannel.appendLine(out)
+            if (out.includes('Swank started at port: 4005')) {
+                attachRepl(state, ctx, { host: 'localhost', port: 4005 })
+            } else if (out.includes('Swank started at port')) {
+                attachRepl(state, ctx)
+            }
+        }
+        const handleDisconnect = (code: number, signal: string) => {
+            if (code === 0) {
+                vscode.window.showInformationMessage('REPL Detached')
+            } else {
+                disconnectAndClearChild(state)
+                vscode.window.showWarningMessage(`Unexpected REPL Detach (signal=${signal} code=${code})`)
```

Comment:
This warning is being shown when the detach command is used. That's an expected detach, so shouldn't show this message.

Reply:
Well this is a hoot.  On Macs, this code behaves, but the only reason I wrote it is so the user would be notified that their connection was gone.  On Windows, it's completely unnecessary because SwankConn actually surfaces a `REPL Connection error: read ECONNRESET`.  I suspect I don't even really need this code if the REPL disconnect was getting picked up on posix operating systems.  Let me see if I can get to the bottom of why the conn doesn't detect the disconnect and display the error.

Reply:
I added some code to the SwankConn to get the REPL to show its connection error on disconnect and removed these warnings since they are unnecessary when the REPL reports its disconnect.